website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -18,6 +18,22 @@ Policy sets are collections of policies you can apply globally or to specific [p
 
 To view and manage policies and policy sets, you must have [manage policy permissions](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policies) for your organization.
 
+## Policy Checks vs Policy Evaluations
```

Comment:
(could be wrong here) The general advice I've had from docs people is to avoid contractions so perhaps just be more descriptive.

```suggestion
## Differences between Policy Checks  and Policy Evaluations
```